catroid/src/main/res/values-bn/strings.xml

```diff
@@ -1716,7 +1716,7 @@
     <string name="formula_editor_logic_comparison">তুলনা অপারেটর</string>
     <string name="formula_editor_object_general">সাধারণ বৈশিষ্ট্য</string>
     <string name="formula_editor_object_movement">গতি বৈশিষ্ট্য</string>
-    <string name="formula_editor_device_visual_detection">Visual sensors</string>
+    <string name="formula_editor_device_face_detection">Visual sensors</string>
```

Comment:
undo this changes, nothing to do here, it will eventually be overwritten by the translations